osmosis-apidb/src/main/java/org/openstreetmap/osmosis/apidb/common/DatabaseContext.java

```diff
@@ -26,6 +26,8 @@
 
     private static final Logger LOG = Logger.getLogger(DatabaseContext.class.getName());
 
+    private static final String UNKNOWN_DATABASE_TYPE = "Unknown database type ";
```

Comment:
I'm okay with this change, although I don't think it adds a lot of value.  I'd prefer to see the entire line replaced with a call to a function that encapsulates the logic.  That would eliminate code duplication which is better than just removing a few duplicated strings.